beast/plotting/plot_param_err.py

```diff
@@ -3,6 +3,7 @@
 from astropy.io import fits
 from matplotlib.colors import LogNorm
 from scipy.stats import binned_statistic_2d as stat2d
+from matplotlib.ticker import (MultipleLocator)
```

Comment:
Very minor.  The () are not needed, especially with one import.

Reply:
Yes! This is leftover from having more ticker imports, and having to remove them to correct code style errors.